entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -18,10 +18,29 @@ If you're using Visual Studio, we recommend the [Package Manager Console tools](
 
 The installation procedure depends on project type and version:
 
+* EF Core 3.x
 * ASP.NET Core version 2.1 and later
 * EF Core 2.x
 * EF Core 1.x
 
+### EF Core 3.x
+
+* `dotnet ef` must be installed as a global or local tool. Most developers will install `dotnet ef` as a global tool with the following command:
+
+  ``` console
+    $ dotnet tool install --global dotnet-ef --version 3.0.0-*
```

Comment:
This will actually not install the right thing when we RTM. I will send another PR.